src/latexify/codegen/function_codegen.py

```diff
@@ -363,6 +366,35 @@ def visit_Call(self, node: ast.Call) -> str:
         arg_strs = [self.visit(arg) for arg in node.args]
         return lstr + ", ".join(arg_strs) + rstr
 
+    def _generate_matrix(self, node: ast.Call) -> str:
+        """Generates a matrix from numpy.matrix."""
+        if len(node.args) != 1:
+            # raise with function name
+            raise ValueError(
+                f"Invalid number of arguments {len(node.args)}"
+            )
+        
+        arg = node.args[0]
+        # TODO: Support string being passed to numpy.matrix.
```

Comment:
Question: what is the "string" in this context?

Reply:
[numpy.matrix ](https://numpy.org/doc/stable/reference/generated/numpy.matrix.html) supports passing in strings like 
and parsing that into a ndarray
```
a = np.matrix('1 2; 3 4')
```